dot/network/sync.go

```diff
@@ -356,6 +358,28 @@ func (q *syncQueue) benchmark() {
 	}
 }
 
+func (q *syncQueue) sentBlockIntervalTelemetry() {
+	for {
+		best, err := q.s.blockState.BestBlockHeader()
+		if err != nil {
+			continue
+		}
+		finalized, err := q.s.blockState.GetFinalizedHeader(0, 0) //nolint
+		if err != nil {
+			continue
+		}
+
+		telemetry.GetInstance().SendBlockIntervalData(&telemetry.BlockIntervalData{
+			BestHash:           best.Hash(),
+			BestHeight:         best.Number,
+			FinalizedHash:      finalized.Hash(),
+			FinalizedHeight:    finalized.Number,
+			TXCount:            0, // todo (ed) determine where to get tx count
```

Comment:
is this the count of txs in a block, or in the queue?

Reply:
I'm not sure what they want reported.